README.md

```diff
@@ -60,14 +60,45 @@ where
 ## Metrics visualisation with Grafana
 
 Metrics stored in the database using `membrane_rtc_engine_timescaledb` can be simply visualized using Grafana.
-To start the dashboard with RTC Engine metrics, mount configuration files, that are in `grafna/provisioning` to the Grafana docker container.
-Eg. if you have cloned this repository in the `/root` directory, add `-v /root/membrane_rtc_engine_timescaledb/grafana/provisioning:/etc/grafana/provisioning` option to your `$ docker run`. Beyond that, you have to set the following environment variables for your docker container
+To start the dashboard with RTC Engine metrics, create a volume with Grafana configuration files that are placed in `priv/grafana/provisioning` and mount it in the Grafana container at the default location: `/etc/grafana/provisioning`. 
+If you release an Elixir project with dependency to `membrane_rtc_engine_timescaledb`, `priv/grafana/provisioning` will be contained in the Docker image. 
```

Comment:
1. Elixir projects are not always released by creating Docker image
2. The important thing here is the info about where that directory will be
3. Move the info about GrafanaHelper here so it is clear what path has to be put in volume's target